control-plane-operator/controllers/awsprivatelink/awsprivatelink_controller.go

```diff
@@ -172,6 +185,10 @@ func (r *AWSEndpointServiceReconciler) SetupWithManager(mgr ctrl.Manager) error
 	awsSession := awsutil.NewSession("control-plane-operator")
```

Comment:
>// AWS_SHARED_CREDENTIALS_FILE and AWS_REGION envvar should be set in operator deployment

Is this a TODO? I can't see where AWS_REGION is set in the cpo deployment reconciler

Reply:
it was done in a previous PR
https://github.com/openshift/hypershift/pull/686/files#diff-b5cde720842b19b53da3f454d827e27327c4559b2e4849906dac728586f51eb3R1965-R1973